packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_InternalExpressionValue.res

```diff
@@ -0,0 +1,260 @@
+module ErrorValue = Reducer_ErrorValue
+module ExternalExpressionValue = ReducerInterface_ExpressionValue
+module Extra_Array = Reducer_Extra_Array
+type internalCode = ExternalExpressionValue.internalCode
+type environment = ExternalExpressionValue.environment
+
+let defaultEnvironment = ExternalExpressionValue.defaultEnvironment
+
+type rec expressionValue =
```

Comment:
it might be good to rename this ``t`` or ``internalExpressionValue``. I don't want both internal and external expression values to both be called ``expressionValue`` in places, it's good to keep them clear.